flow.c

```diff
@@ -1363,7 +1358,7 @@ static u32 FlowDump(u8* OutputStr, u64 TS, FlowRecord_t* Flow, u32 FlowID)
 		H.FirstTS		= Flow->FirstTS;
 		H.TotalPkt		= Flow->TotalPkt;
```

Comment:
Please add 8bit field to HistogramDump_t so there is a 1bit field for

VLAN0
VLAN1
VLAN2


MPLS0
MPLS1
MPLS2

remaining bits free

if the flow has vlan or mpls set, its recorded here. Its important when generating the flows on the other end to capture the signature of the flow more accurately. As we are trying to replicate the decoding performance.